src/yalexs_ble/push.py

```diff
@@ -811,6 +818,53 @@ async def validate(self) -> None:
         await self._update()
         _LOGGER.debug("%s: Finished validate", self.name)
 
+    async def _poll_battery(
```

Comment:
This solution makes sense to me, however we should add some test coverage for it